client-js/app/elements/labrad-manager.ts

```diff
@@ -1,9 +1,14 @@
-import {ConnectionInfo} from '../scripts/manager';
+import {ConnectionInfo, ManagerApi} from '../scripts/manager';
 
 @component('labrad-manager')
 export class LabradManager extends polymer.Base {
 
   @property({type: Array, notify: true})
   connections: Array<ConnectionInfo>;
 
+  mgr: ManagerApi;
+
+  closeConnection(event) {
+    this.mgr.connectionClose(event.currentTarget.connId);
```

Comment:
I like `window.confirm` a lot compared to messing around with polymer dialogs, though that probably says more about polymer dialogs...